contrib/ci/configure.sh

```diff
@@ -29,62 +29,34 @@ declare -a CONFIGURE_ARG_LIST=(
     "--enable-ldb-version-check"
     "--with-syslog=journald"
     "--enable-systemtap"
-    "--with-python2-bindings"
+    "--without-python2-bindings"
 )
 
 
-if [[ "$DISTRO_BRANCH" == -redhat-redhatenterprise*-6.*- ||
-      "$DISTRO_BRANCH" == -redhat-centos-6.*- ]]; then
-    CONFIGURE_ARG_LIST+=(
-        "--with-smb-idmap-interface-version=5"
-        "--disable-cifs-idmap-plugin"
-        "--with-syslog=syslog"
-        "--without-python3-bindings"
-        "--without-kcm"
-    )
-fi
-
-if [[ "$DISTRO_BRANCH" == -redhat-fedora-2[0-2]* ]]; then
-    CONFIGURE_ARG_LIST+=(
-        "--without-kcm"
-    )
-fi
-
-if [[ "$DISTRO_BRANCH" == -redhat-redhatenterprise*-7.*- ||
-      "$DISTRO_BRANCH" == -redhat-centos-7.*- ]]; then
-    CONFIGURE_ARG_LIST+=(
-        "--without-python3-bindings"
-    )
-fi
-
 # Different versions of Debian might need different versions here but this is
 # sufficient to make the CI work
 if [[ "$DISTRO_BRANCH" == -debian-* ]]; then
     CONFIGURE_ARG_LIST+=(
-        "--without-python2-bindings"
         "--with-smb-idmap-interface-version=5"
     )
 fi
 
-if [[ "$DISTRO_BRANCH" == -redhat-fedora-4[0-9]* ||
-      "$DISTRO_BRANCH" == -redhat-fedora-3[2-9]* ||
-      "$DISTRO_BRANCH" == -redhat-centos*-9*- ||
-      "$DISTRO_BRANCH" == -redhat-redhatenterprise*-9.*- ]]; then
-    CONFIGURE_ARG_LIST+=(
-        "--without-python2-bindings"
-    )
-fi
-
-if [[ "$DISTRO_BRANCH" == -redhat-fedora-3[5-9]* ||
-      "$DISTRO_BRANCH" == -redhat-redhatenterprise*-9.*- ]]; then
+if [[ "$DISTRO_BRANCH" == -redhat-fedora-3[7-9]* ||
+      "$DISTRO_BRANCH" == -redhat-fedora-4[0-9]* ||
```

Comment:
This could be simplified to `-redhat-fedora-*`

Reply:
Done